src/App.tsx

```diff
@@ -15,48 +21,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+interface PreparedGoodsProps {
+  sortType: SortType | string;
+  reverse: boolean;
+}
+
+function getPreparedGoods(
+  goods: string[],
+  { sortType, reverse }: PreparedGoodsProps,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortType) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.Alphabetically:
+          return good1.localeCompare(good2);
+
+        case SortType.ByLength:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return reverse ? preparedGoods.reverse() : preparedGoods;
+}
+
+export const App = () => {
+  const [sortType, setSortType] = useState('');
+  const [reverse, setReverse] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/96409563/f53be370-cfc5-484d-8a6b-7d568e8d1d19)